pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -778,6 +786,44 @@ func validateNamedRuleWithOperations(n *admissionregistration.NamedRuleWithOpera
 	return allErrors
 }
 
+func validateMatchCondition(v *admissionregistration.MatchCondition, fldPath *field.Path) field.ErrorList {
+	var allErrors field.ErrorList
+	trimmedExpression := strings.TrimSpace(v.Expression)
+	if len(trimmedExpression) == 0 {
+		allErrors = append(allErrors, field.Required(fldPath.Child("expression"), "expression is not specified"))
+	} else {
+		result := plugincel.CompileCELExpression(
+			&generic.MatchCondition{
+				Expression: trimmedExpression,
+			},
+			plugincel.OptionalVariableDeclarations{
+				HasParams:     false,
+				HasAuthorizer: false,
+			},
+			celconfig.PerCallLimit)
+		if result.Error != nil {
+			switch result.Error.Type {
+			case cel.ErrorTypeRequired:
+				allErrors = append(allErrors, field.Required(fldPath.Child("expression"), result.Error.Detail))
+			case cel.ErrorTypeInvalid:
+				allErrors = append(allErrors, field.Invalid(fldPath.Child("expression"), v.Expression, result.Error.Detail))
+			case cel.ErrorTypeInternal:
+				allErrors = append(allErrors, field.InternalError(fldPath.Child("expression"), result.Error))
+			default:
+				allErrors = append(allErrors, field.InternalError(fldPath.Child("expression"), fmt.Errorf("unsupported error type: %w", result.Error)))
+			}
+		}
+	}
+	if len(v.Name) == 0 {
+		allErrors = append(allErrors, field.Invalid(fldPath.Child("name"), v.Name, "name must be specified"))
```

Comment:
This should be `field.Required` (and you can leave off the message, same as above)

Reply:
done